src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -221,3 +217,32 @@ def tearDown(self):
                 server_utils.stop_server(hosts=self.hostlist_servers)
             finally:
                 super(TestWithServers, self).tearDown()
+
+    def start_servers_and_clients(self, server_groups=None):
```

Comment:
I think we should simplify this down to just line 238-248 and call it `start_servers`.  Lines 228-236 should then go back into the `setup()` method.